documentation/next-generation-quickstart.md

```diff
@@ -0,0 +1,428 @@
+
+Getting Started - the next generation of Azure SDK for Javascript/Typescript
+=============================================================
+
+We are excited to announce that a new set of management libraries are
+now production-ready. Those packages share a number of new features
+such as Azure Identity support, HTTP pipeline, error-handling.,etc, and
+they also follow the new Azure SDK guidelines which create easy-to-use
+APIs that are idiomatic, compatible, and dependable.
+
+Currently, we have previewed several packages such as `azure/arm-resources`, `@azure/arm-storage`, 
+`@azure/arm-compute`, `@azure/arm-network` for next-generation. See more from npmjs.com and find 
+the latest version under `next` tag and have a try.
+
```

Comment:
add a section that links to the migration guide you wrote

If you are interested in upgarding to the latest new generation of SDK, please refer to this [migration guide]